.github/workflows/pr.yaml

```diff
@@ -20,8 +21,10 @@ jobs:
         run: docker build -t civiform-dev --cache-from docker.io/civiform/civiform-dev:latest ./
       - name: Run tests
         run: USE_LOCAL_CIVIFORM=true bin/run-test-ci
+
   run_browser_tests_aws:
     runs-on: ubuntu-latest
+    if: contains(toJSON(steps.file_changes.outputs.files), 'browser-test/') || contains(toJSON(steps.file_changes.outputs.files), 'server/')
```

Comment:
These should also run when the bin scripts that they run change, or when the docker files change

Reply:
Done